feat(contrib)!: remove deprecated `minijnja` module (#4358)
litestar-org/litestar: docs/release-notes/changelog.rst

        Remove the deprecated ``litestar.contrib.pydantic`` module. Code still using imports
        from this module should switch to using ``litestar.plugins.pydantic``.

    .. change:: Remove deprecated module ``litestar/contrib/minijnja``
        :type: feature
        :breaking:
        :pr: 4358
        :issue: 4358

>         Remove the deprecated module ``litestar/contrib/minijnja``.

Author: @provinzkraut Added a clarification. Ty

        :pr: 4358
        :issue: 4358

>         Remove the deprecated module ``litestar.contrib.minijnja``. Module was created with a typo,
>         the typo module is removed. Instead ``litestar.contrib.minijinja`` should be used.

Reviewer: ```suggestion
        Remove the deprecated module ``litestar.contrib.minijnja``. This module was created with a typo in its name (`minijnja` instead of `minijinja`). Instead ``litestar.contrib.minijinja`` should be used.
```
